src/objects.c

```diff
@@ -2494,11 +2494,19 @@ static CK_RV prep_rsa_find(P11PROV_CTX *ctx, const OSSL_PARAM params[],
     return CKR_OK;
 }
 
+#define PUB_KEY_SIZE 200
```

Comment:
This looks odd, is this max length for all curves?
If so maybe rename to MAX_EC_PUB_KEY_SIZE